invenio_users_resources/services/users/config.py

```diff
@@ -68,9 +68,14 @@ class UserSearchOptions(SearchOptions, SearchOptionsMixin):
     # The user search needs to be highly restricted to avoid leaking
     # account information, hence do not edit here unless you are
     # absolutely sure what you're doing.
-    suggest_parser_cls = SuggestQueryParser.factory(
+    suggest_parser_cls = CompositeSuggestQueryParser.factory(
         tree_transformer_cls=SearchFieldTransformer,
-        fields=["username^2", "email^2", "profile.full_name^3", "profile.affiliations"],
+        fields=[
+            "username^2",
+            "email.keyword^2",
```

Comment:
`email` had to be changed to `email.keyword` for the tests to pass.

Reply:
Minor: Maybe we could keep the email text field as well, might be useful, wdyt?
```suggestion
            "email.keyword^2",
            "email^2",
```

Reply:
@ptamarit what about @jrcastro2 suggestion?

Reply:
@jrcastro2 Suggestion applied, thanks.